react/ninja-react/src/components/UserRegistrationForm.jsx

```diff
@@ -239,9 +235,18 @@ export class UserRegistrationForm extends React.Component {
               clearFormOnSubmit: true
             });
           }
+          //TODO: add in here checking for specific errors, e.g. LDAP lookup fad, trello/github username not found
+        }
+        //undefined error response == network error
+        else {
+          this.setState({
+            showModal: true,
+            modalTitle: "Network Error",
+            modalText:
+              "The backend server is down, or your network connection isn't working.",
+            clearFormOnSubmit: true
```

Comment:
Do we want to clear the form when an error occurs? I personally say no. Thoughts?

Reply:
Agreed